examples/life.htm

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+	<style>
+		.flatbutton
+		{
+			width:		200px;
+			margin:		0 auto;
+			text-align:	center;
+			font-family:	Arial;
+			font-size:	16px;
+			font-weight:	bold;
+			cursor:		pointer;
+		}
+	</style>
+	<head>
+		<title>Life via qKit</title>
+	</head>
+	<body>
+		<script src="qkit.js"></script><!-- https://raw.githubusercontent.com/fahadhaidari/qKit/master/qkit.js -->
+		<script src="life.js"></script>

```

Comment:
To give more guidance, `life.js` should be hosted in `index.html` file in the root directory.
The file `life.js` itself should be added in the `examples` directory

Reply:
Will comply.  As a side-note, the idea behind the HTM was to provide an interface to showcase features built in to the JS.  Do you have any interest in a convention or annex for that?  Surely there'd be a traffic-jam if each JS came with its variety of index.html.